cloud-prepare/vpc-peering.md

```diff
@@ -0,0 +1,210 @@
+# Submariner VPC Peering Support
+
+This document describes the technical details and impact of our proposal design
+to include Virtual Private Cloud (VPC) Peering support for Submariner and
+`subctl` located at this [Pull Request](https://github.com/submariner-io/cloud-prepare/pull/190).
+
+## Abstract
+This idea was born meanwhile the CockroachDB project focused to deploy a
+multi-cluster setup of CockroachDB and communicate the different clusters across
+regions using Submariner.
+
+One of the requirements generated from that project was to deploy CockroachDB on
```

Comment:
One may wonder why Submariner is even needed if something like VPC Peering is in place. Doesn't the underlying cloud provide L3 routing between the clusters on top of the peering connection? If so, the case for using Submariner might be limited to Lighthouse (DNS) only? @sridhargaddam can you comment on that?

Reply:
With the peering, only the nodes of two Kubernetes clusters will be reachable.  We still need a component like Submariner. for the pods/service of two Kubernetes clusters to be reachable,

Reply:
Yes, I think we should explicitly mention that in the design doc, i.e what is the value of Submariner in this type of configuration.